Scarb.toml

```diff
@@ -3,11 +3,11 @@ name = "resolver"
 version = "0.1.0"
 
 [dependencies]
-starknet = "2.3.1"
-naming = { git = "https://github.com/starknet-id/naming.git", rev = "6e45f7fc2bc9e8e9d874a1d2c2d46e3226a96f6d" }
-openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", rev = "f3e2a5f0547a429c716f32471b06df729cbdfb9f" }
-identity = { git = "https://github.com/starknet-id/identity.git", rev = "eb37846330b78e1410cf5e3e17a5dcca5652f921" }
-storage_read = { git = "https://github.com/starknet-id/storage_read_component.git", rev = "c6c69e15d34abfc39ac51dc21b96724e2e19ff31" }
+starknet = "2.6.3"
+naming = { git = "https://github.com/starknet-id/naming.git", rev = "7fa861a7a6e56c96ba3b508c66d7587d2597385a" }
+openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", rev = "0697004db74502ce49900edef37331dd03531356" }
+identity = { git = "https://github.com/starknet-id/identity.git", rev = "2185e542838c639be5cee47979e2462822000fd0" }
```

Comment:
Not building because here you didn't import the same version of the identity package as the one inside the naming package. It should be `5ae7fafdd20abf4b1e789b83af5db7e23ad39a7c`